tests/e2e/vsan_stretched_cluster.go

```diff
@@ -1638,9 +1728,15 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 			}
 		}()
 
+		if rwxAccessMode {
+			command = execRWXCommandPod
```

Comment:
move this to before each and give a proper review comment as what this if/esle is doing